sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FlatMapElementsTest.java

```diff
@@ -103,6 +106,19 @@ public void testFlatMapFnOutputTypeDescriptor() throws Exception {
   }
 
   @Test
+  public void testSimpleFunctionClassDisplayData() {
+    SimpleFunction<Integer, List<Integer>> simpleFn = new SimpleFunction<Integer, List<Integer>>() {
+      @Override
+      public List<Integer> apply(Integer input) {
+        return Collections.emptyList();
+      }
+    };
```

Comment:
Can you update these tests to also verify display data items from the SimpleFunction are registered? i.e., add a `populateDisplayData()` override to add some display data and make sure it gets wired through.